src/acl/Acl.h

```diff
@@ -62,11 +62,11 @@ class ACL
     /// \returns (linked) Options supported by this ACL
     virtual const Acl::Options &options() { return Acl::NoOptions(); }
 
-    /// configures ACL options, throwing on configuration errors
+    /// configures supported ACL options, throwing on configuration errors
     virtual void parseFlags();
 
     /// parses node representation in squid.conf; dies on failures
-    virtual void parse() = 0;
+    virtual void parse() {}
```

Comment:
I think this change should be undone. AFAICT, it was a result of two changes, both reasonable, but the end result (i.e. the nothing-to-parse default) seems like rather odd to me. If we only have a few ACLs that have nothing to parse (e.g., Acl::ConnectionsEncrypted), then I would rather force them to say so explicitly (by providing an empty parse() method) than risk an ACL kid that _forgets_ to declare a parse() method or, even more likely, accidentally _loses_ one.